README.md

```diff
@@ -1,62 +1,26 @@
-
-Your Access-Token server should provide a valid token for valid credentials by URL:
-
- ```
-$REACT_APP_ACCESS_TOKEN_SERVICE_URL?identity=<USER_PROVIDED_USERNAME>&password=<USER_PROVIDED_PASSWORD>
- ```
-
-And return HTTP 401 in case of invalid credentials.
-
-If an `.env` file doesn't exist, create one with the following contents:
-```
-REACT_APP_ACCESS_TOKEN_SERVICE_URL=http://example.com/get-token/
-```
-
-NOTE: No need for quotes around the URL, they will be added automatically.
+What you'll need to get started:
+- A [fork](https://github.com/twilio/twilio-conversations-demo-react/fork) of this repo to work with.
+- A way to [generate Conversations access tokens](#generating-access-tokens).
+- Optional: A Firebase project to [set up push notifications](#setting-up-push-notifications).
 
-### Save Firebase project configuration to `firebase-config.js` file in the `public` folder
+## Generating Access Tokens
+Client apps need [access tokens](https://www.twilio.com/docs/conversations/create-tokens) to connect to authenticate and connect to the Conversations service as a user. These tokens should be generated by your backend server using your private Twilio API Keys. If you already have a service that does this, skip to [setting the token service URL](#set-the-token-service-url). 
 
-It should contain definition of the `firebaseConfig` variable like in the code below:
+For testing purposes, you can quikly set up a Twilio Serverles Functions to generate access tokens. Note that this is not a product ready implementation.
 
-```javascript
-var firebaseConfig = {
-  apiKey: "sample__key12345678901234567890",
-  authDomain: "convo-demo-app-internal.firebaseapp.com",
-  projectId: "convo-demo-app-internal",
-  storageBucket: "convo-demo-app-internal.appspot.com",
-  messagingSenderId: "1234567890",
-  appId: "1:1234567890:web:1234abcd",
-  measurementId: "EXAMPLE_ID"
-};
-```
-Read more about [Firebase project configuration](https://firebase.google.com/docs/web/learn-more#config-object).
-
-### For testing purposes, it is possible to create a simple token generator using a Twilio function:
-
-1. Create a new function in [Twilio functions](https://www.twilio.com/console/functions/manage) using template `Blank`
-2. On the next, line add `/token-service` to the `PATH`. Copy the whole `PATH` and use it as `REACT_APP_ACCESS_TOKEN_SERVICE_URL` (see above)
-3. Uncheck the **Check for valid Twilio signature** checkbox
-4. Insert the following code:
+### Generating Access Tokens with Twilio Functions
+1. Create a [Twilio Functions Service from the console](https://www.twilio.com/console/functions/overview) and add a new function using the **Add+** button.
+3. Set the function path name to `/token-service`, then **Copy URL** from the "kebab" three dot menu next to it and and use it as `REACT_APP_ACCESS_TOKEN_SERVICE_URL` environment variable (see above).
```

Comment:
There's not a single word about REACT_APP_ACCESS_TOKEN_SERVICE_URL above this line. Change to `(see below)`.